src/lib/createInstanceManager.js

```diff
@@ -10,11 +10,11 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-const getInstanceNameByPropertyID = (accounts, propertyID) => {
+const getInstanceNameByPropertyId = (accounts, propertyId) => {
   let matchingInstanceName;
   for (let i = 0; i < accounts.length; i += 1) {
     const account = accounts[i];
-    if (account.propertyID === propertyID) {
+    if (account.propertyId === propertyId) {
       matchingInstanceName = account.instanceName;
       break;
     }
```

Comment:
Can we simplify it to one line like
```
matchingInstance = accounts.find(account => account.propertyId === propertyId)
```

Reply:
Yes! Good catch. I don't know what I was thinking.

Reply:
Actually, I do know what I was thinking. I can't use `find` because it isn't supported in Internet Explorer, so I have to do the iteration myself. I could do a `forEach`, but I couldn't escape the iteration once the account is found, so I opted for the for loop.